### ISSUE TITLE
need help to install ghostscript

### ISSUE THREAD

**Issue by hooman-pro** [spatie/pdf-to-image#141]
when i use your library and installed imagick 
before install ghostscript i get this error:
PDFDelegateFailed `The system cannot find the file specified. ' @ error/pdf.c/ReadPDFImage/794f
but when i install ghostscript 64 bit chorome says:
This site can’t be reachedThe connection was reset.
Try:

Checking the connection
Checking the proxy and the firewall
Running Windows Network Diagnostics
ERR_CONNECTION_RESET 
!!!!
I use windows 10 and xampp apache. after install gostscript

in your ducument there is part " Issues regarding Ghostscript" but it is for nginx 
what should i do in apache that have not php-fpm.conf file

**Comment by hooman-pro** [comment]
from command line or from exec by php this command:
magick -density 1200 c:/a/a.pf c:/a/a.jpg 
work currectly . but by imagick class or your library not work and when run url google chorom say : site cant be reachabed .....

**Comment by freekmurze** [comment]
Troubleshooting imagick and/or Ghostscript is out of scope for this issue tracker.

**Comment by beard7** [comment]
I had a similar nightmare trying to get imagick and Ghostscript to behave on Windows. 

I fixed this issue: **PDFDelegateFailed `The system cannot find the file specified. ' @ error/pdf.c/ReadPDFImage/794** by simply renaming gswin64c.exe to gs.exe

Then I got a different error: **PDFDelegateFailed `Error: /undefinedfilename**
Which turned out to be because Ghostscript couldn't read the temporary files in C:\Windows\temp. As a quick fix, I simply granted Everyone read/write access to this folder (until I come up with a better solution).

Now all is working. Hope this helps.
